src/pykeen/trackers.py

```diff
@@ -85,3 +86,44 @@ def log_params(self, params: Dict[str, Any], prefix: Optional[str] = None) -> No
 
     def end_run(self) -> None:  # noqa: D102
         self.mlflow.end_run()
+
+
+class WANDBResultTracker(ResultTracker):
+    """A tracker for Weights & Biases. Note that you have to perform wandb login beforehand."""
+
+    def __init__(
+            self,
+            project_name: str,
+            experiment_name: Optional[str] = None,
+    ):
+        """
+        Initialize result tracking via WANDB.
+
+        :param project_name:
+            project name your wandb login has access to.
+        :param experiment_name:
+            The experiment name to appear on the website. If not given, WANDB will generate a random name.
+        """
+        import wandb as _wandb
+        self.wandb = _wandb
+        assert project_name is not None, "Weights & Biases requires a project name."
+        self.project_name = project_name
+
+        if experiment_name is not None:
+            self.wandb.init(project=self.project_name, name=experiment_name)
+        else:
+            self.wandb.init(project=self.project_name)
+
+    def log_metrics(
+        self,
+        metrics: Dict[str, float],
+        step: Optional[int] = None,
+        prefix: Optional[str] = None,
+    ) -> None:  # noqa: D102
+        metrics = flatten_dictionary(dictionary=metrics, prefix=prefix)
+        self.wandb.log(metrics)
+
+    def log_params(self, params: Dict[str, Any], prefix: Optional[str] = None) -> None:  # noqa: D102
+        params = flatten_dictionary(dictionary=params, prefix=prefix)
+        for k, v in params.items():
+            self.wandb.config[k] = v
```

Comment:
this is a dict? you can just do

```python
self.wand.config.update(params)
```

Reply:
Good call, done in 6ff2ef2fbc7b7a9ba51dc003d39d32589237dc7f